components/expenses/ExpenseForm.js

```diff
@@ -302,6 +302,12 @@ const ExpenseFormBody = ({
     }
   }, [values.payeeLocation]);
 
+  React.useEffect(() => {
+    if (dirty) {
+      formik.setFieldValue('currency', undefined);
+    }
+  }, [values.payoutMethod]);
+
```

Comment:
Don't we want to only reset if the currency changes with something else? What if it's a collective without multi-currency enabled, sound like they won't be able to set it back?

Reply:
Ops, yes I've added a check to see if it's multi-currency enabled collective so that it doesn't break for that case.